<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_DialogSetConf</class>
 <widget class="QDialog" name="UI_DialogSetConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Configure</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>71</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>PORT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Web Path</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_IP3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>71</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_IP2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>71</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_IP1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>71</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_IP0">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>71</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Port">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>80</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_WebPath">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Msg">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>190</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_OK">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
